bc_obps/compliance/models/rls_configs/compliance_period.py

```diff
@@ -0,0 +1,19 @@
+from compliance.enums import ComplianceTableNames
+from rls.enums import RlsRoles, RlsOperations
+from rls.utils.helpers import generate_rls_grants
+
+
+class Rls:
+    role_grants_mapping = {
+        RlsRoles.INDUSTRY_USER: [
+            RlsOperations.SELECT,
+            RlsOperations.INSERT,
+            RlsOperations.UPDATE,
+            RlsOperations.DELETE,
+        ],
```

Comment:
The compliance period is determined by the program, industry users shouldn't be able to edit it